internal/service/networkfirewall/rule_group_test.go

```diff
@@ -56,6 +56,41 @@ func TestAccNetworkFirewallRuleGroup_Basic_rulesSourceList(t *testing.T) {
 	})
 }
 
+func TestAccNetworkFirewallRuleGroup_Basic_referenceSets(t *testing.T) {
```

Comment:
Please consider adding an 'update' test to ensure the resource is updated properly

Reply:
Added Update Test as requested.